rasa/nlu/tokenizers/convert_tokenizer.py

```diff
@@ -32,24 +33,25 @@ def _load_tokenizer_params(self):
         # needed to load the ConveRT model
         import tensorflow_text
         import tensorflow_hub as tfhub
+        import os
 
-        self.graph = tf.Graph()
         model_url = "http://models.poly-ai.com/convert/v1/model.tar.gz"
 
-        with self.graph.as_default():
-            self.session = tf.Session()
-            self.module = tfhub.Module(model_url)
+        # required to take care of cases when other files are already
+        # stored in the default TFHUB_CACHE_DIR
+        try:
+            self.module = tfhub.load(model_url)
```

Comment:
duplicated code, the loading code should go into a function

Reply:
moved it to a function.